internal/controllers/vgmanager/dmsetup/dmsetup.go

```diff
@@ -43,11 +44,15 @@ func (dmsetup *HostDmsetup) Remove(deviceName string) error {
 	args = append(args, deviceName)
 	output, err := dmsetup.ExecuteCommandWithOutputAsHost(dmsetup.dmsetup, args...)
 	if err != nil {
-		if strings.Contains(output, "not found") {
-			return ErrReferenceNotFound
-		}
 		return fmt.Errorf("failed to remove the reference from device-mapper %q: %v", deviceName, err)
 	}
+	data, err := io.ReadAll(output)
```

Comment:
Should this be inside the err check like the deleted part?

Reply:
Not sure what you mean by that, could explain?